### ISSUE TITLE
Search functionality not working

### ISSUE THREAD

**Issue by HoChihchou** [elrumo/macOS_Big_Sur_icons_replacements#1517]
Is this my problem?
![CleanShot 2024-11-30 at 20 57 39@2x](https://github.com/user-attachments/assets/42472232-c615-420e-bdfe-3eb6393478be)

**Comment by xywyzy** [comment]
Yes, I agree. It seems we can no longer search with uploader ID, and instead, we need to use the API for downloads. I am concerned the website may soon introduce premium features. Download as many icons as you can before things get different.

**Comment by UserC2** [comment]
The pinwheel beside the "X" indicates the search hasn't completed yet. Wait for the search to complete and you'll see results.
